styles/components/panel.scss

```diff
@@ -44,6 +44,23 @@
     border-bottom-right-radius: $border-radius-base;
   }
 
+  > .table.-bordered > thead > tr > th:first-child, 
+  > .table.-bordered > tbody > tr > td:first-child {
+    border-left: 0;
+  }
+
+  > .table.-bordered > thead > tr > th:last-child, 
+  > .table.-bordered > tbody > tr > td:last-child {
+    border-right: 0;
+  }
+
+  > .table.-bordered > tbody > tr:last-child { border-bottom: 0; }
```

Comment:
I don't think it's necessary to include the modifier `-bordered` here. What do you think?